recipes/cccl/all/test_package/conanfile.py

```diff
@@ -9,25 +9,20 @@ class CcclTestConan(ConanFile):
     test_type = "explicit"
     win_bash = True
 
-    @property
-    def _settings_build(self):
-        # TODO: Remove for Conan v2
-        return getattr(self, "settings_build", self.settings)
-
     def build_requirements(self):
         self.tool_requires(self.tested_reference_str)
-        if self._settings_build.os == "Windows" and not self.conf.get("tools.microsoft.bash:path", default=False, check_type=bool):
+        if self.settings_build.os == "Windows" and not self.conf.get("tools.microsoft.bash:path", default="", check_type=str):
```

Comment:
in most other places in the repo I think the `default=""` is not there - the default is None, so it would have the same effect. e.g in other places:
```
            if not self.conf.get("tools.microsoft.bash:path", check_type=str):
                self.tool_requires("msys2/cci.latest")
```
(minor, otherwise)

Reply:
Done, removed in the commit [c3ba872](https://github.com/conan-io/conan-center-index/pull/26874/commits/c3ba87288945cfdf3abff69d54c8add039655472)

Reply:
Sorry, I understood another thing ... gonna revert it and simplify.

Reply:
Done, fixed by the commit [2558e20](https://github.com/conan-io/conan-center-index/pull/26874/commits/2558e20905965fae824d8b768335122f64557d2d)